integration_tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Integration Tests
```

Comment:
I think we could avoid the term "app" and just use integration/implementation test. Also worth linking from the main `README`, as you mentioned.

Otherwise, this is good documentation and gets across the intent of the tests really well.